policy/modules/admin/tomoyo-auditd.if

```diff
@@ -0,0 +1,27 @@
+## <summary>Audit log daemon for TOMOYO Linux MAC.</summary>
+
+########################################
+## <summary>
+##	All of the rules required to
+##	administrate tomoyo-auditd.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`tomoyo_auditd_admin',`
+	gen_require(`
+		type tomoyo_auditd_t;
```

Comment:
you are referencing tomoyo_auditd_t here but you arent actually using it. You may want to allow your tomoyo-auditd admin to send some signals to it and to ps it

Reply:
Added ps and signal_perms